File: JAVASCRIPT/beginner/1049.js
//https://www.urionlinejudge.com.br/judge/en/problems/view/1049
var input = require('fs').readFileSync('./input.txt', 'utf8');
var lines = input.split('\n');

const [vertebracao, tipo, alimentacao] = lines;

if (vertebracao === "vertebrado") {
    if (tipo === "ave") {
        if (alimentacao === "carnivoro") {
            console.log("aguia")
        } else {
            console.log("pomba")
        }
    } else {
        if (alimentacao === "onivoro") {
            console.log("homem")
        } else {
            console.log("vaca")
        }
    }
} else {
    if (tipo === "inseto") {
        if (alimentacao === "hematofago") {
            console.log("pulga")
        } else {
            console.log("lagarta")
        }
    } else {
        if (alimentacao === "hematofago") {
            console.log("sanguessuga")
        } else {
            console.log("minhoca")
        }
    }
}

File: C99/beginner/1049.c
#include <stdio.h>

int main() {
  char firstInput[20];
  char secondInput[20];
  char thirdInput[20];
  int i, isVertebrado;

  scanf("%s", firstInput);
  
  if (firstInput[0] == 'v') {
    scanf("%s", secondInput);
    if (secondInput[0] == 'a') {
    scanf("%s", thirdInput);
      if (thirdInput[0] == 'c') {
        printf("aguia\n");
      } else {
        printf("pomba\n");
      }
    } else {
    scanf("%s", thirdInput);
      if (thirdInput[0] == 'o') {
        printf("homem\n");
      } else {
        printf("vaca\n");
      }
    }
  } else {
    scanf("%s", secondInput);
    if (secondInput[0] == 'i') {
    scanf("%s", thirdInput);
      if (thirdInput[2] == 'm') {
        printf("pulga\n");
      } else {
        printf("lagarta\n");
      }
    } else {
    scanf("%s", thirdInput);
      if (thirdInput[0] == 'h') {
        printf("sanguessuga\n");
      } else {
        printf("minhoca\n");
      }
    }
  }

  return 0;
}